crates/xilem_html/src/diff.rs

```diff
@@ -1,5 +1,6 @@
 use std::{cmp::Ordering, collections::BTreeMap, iter::Peekable};
 
+#[allow(unused)]
```

Comment:
I assume this is maybe unused now because it is no longer used by the other files and only by tests?

Not sure what we should be keeping, exposing, etc.

Reply:
Yes it is/was, I have refactored it into your suggestion below (`diff_kv_iterables`) and used it in the `VecMap::diff` method.